src/components/Menu/MenuItem/MenuToggle.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import {ReactComponent as CloseIcon} from "assets/icon-close.svg";
+import './MenuItem.scss';
+
+type MenuToggleProps = {
+    children: React.ReactNode;
+    direction: 'left' | 'right';
+    index: number;
+    onToggle?: (active: boolean) => void;
+    toggleStartLabel: string;
+    toggleStopLabel: string;
+}
+
+function MenuToggle(props: MenuToggleProps) {
+
+    const [status, setStatus] = useState(false);
+
+    return (<button 
+                className={`menu-item menu-item${props.index} menu-item${status ? '--active' : '--disabled'} menu-item--${props.direction}`}
+                onClick={_ => setStatus(prevValue => !prevValue)}
+            >
+        <div className='menu-item__tooltip'>
+            <span className='tooltip__text'>
+                {status ? props.toggleStopLabel : props.toggleStartLabel}
+            </span>
+        </div>
+        <CloseIcon className='menu-item__icon menu-item__icon--end'/>
+        {props.children}
```

Comment:
I would suggest to pass the icon as a prop and not within the `children`. It makes it more explicit and maybe you can even start to restrict the usage of the prop with TypeScript

Reply:
Done ✅